src/webfaf/problems.py

```diff
@@ -640,3 +640,4 @@ def bthash_forward(bthash=None):
                                problems=problems)
 
     abort(404)
+    return None
```

Comment:
The `return None` will never be executed because the `abort(404)` will redirect you to an error page before it.

Reply:
I know, but pylint doesn't. There are two options - this return or disable pylint warning on this function and I found this less harmful

Reply:
You can change it to `return abort(404)`, that should stop pylint complaining.